### DISCUSSION TITLE
Using SQLite with SQLCipher, passing instance?

### DISCUSSION THREAD

**Discussion by appinteractive** [prisma/prisma#4233] [Q&A] [answered]
Hey there 👋, 

I'm trying to use Prisma with an encrypted SQLite Database by using [SQLCipher](https://github.com/journeyapps/node-sqlcipher). However, therefore I would need to use that specific package as the sqlite instance but I can't pass an instance to the configuration which has the extension build in. 

So is there a way that Prisma would use the correct SQLite instance with that extension? As a reference, I try to use it in the context of an electron app.

Thanks in advance

**Comment by PrimeDominus** [comment]
I would also like some suggestions for this, or for using Prisma with [sqleet](https://github.com/resilar/sqleet) or any other options for encrypting sqlite databases with Prisma

**Reply by appinteractive** [reply]
Yeah sqleet might be better as it officially "sucks less" then sqlcipher, at least from the license aspect.

**Reply by PrimeDominus** [reply, marked as answer]
This may help formulate a way to get Prisma to work with sqleet and nodejs https://github.com/LaurentDhont/better-sqlite3-sqleet

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
